src/main/groovy/jenkins/automation/builders/JsJobBuilder.groovy

```diff
@@ -17,7 +17,7 @@ class JsJobBuilder {
         fingerprint()
         defaultExcludes()
     }
-    List<String> emails
+    def emails
```

Comment:
We want to be able to pass either a list or string for emails. This makes it dynamic variable. It will use "duck-typing" at runtime.